### DISCUSSION TITLE
How to get job.status into shell script?

### DISCUSSION THREAD

**Discussion by vitalykarasik** [community/community#25729] [Actions]
<p>It should be really basic stuff but I couldn’t figure it out…<br>
There is job.status context, and I want to use it in my shell script (in order to set a color for Slack notification). How can I get this  job.status into script?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You need to provide it to the step in some way. Exactly how depends on how your script wants to consume that data. You could put it on the command line:</p>
<pre><code class="lang-yaml">steps:
  - run: ./my_script.sh ${{ job.status }}
</code></pre>
<p>Or into the environment:</p>
<pre><code class="lang-yaml">steps:
  - run: ./my_script.sh
    env:
      STATUS: ${{ job.status }}
</code></pre>

**Comment by vitalykarasik** [comment]
<p>Many thanks!<br>
In fact, I asked the wrong question. I’m using Github action for Slack notification and I’d like to use the job status for setting Slack notification color. I don’t like to idea using separate steps for each job status, i.e. for success(), failure(), and so on.<br>
I’m trying to use an expression for setting color param; without success so far.</p>
<pre><code class="lang-auto"> with:
      color: ${{ env.STATUS }} == 'success' &amp;&amp; 'good' || 'bad'
</code></pre>
<p>Tried “color: failure() &amp;&amp; ‘good’ || ‘bad’” and other variations  - the same result.</p>

**Comment by vitalykarasik** [comment]
<p>I figured it out - as usually RTFM helps.<br>
Thank you again.</p>
<p>color: ${{ job.status == ‘success’ &amp;&amp; ‘good’ || ‘bad’ }}</p>
